working-groups.md

```diff
@@ -215,6 +220,20 @@ Tekton projects.
 | <img width="30px" src="https://github.com/chhsia0.png">    | Chun-Hung Hsiao   | D2iQ    | [chhsia0](https://github.com/chhsia0)       |
 | <img width="30px" src="https://github.com/pritidesai.png"> | Priti Desai       | IBM     | [pritidesai](https://github.com/pritidesai) |
 
+### Australia + North America
+
+| Artifact                   | Link                       |
+| -------------------------- | -------------------------- |
+| Community Meeting VC       | [https://meet.google.com/nyw-cwme-iia](https://meet.google.com/nyw-cwme-iia) |
+| Community Meeting Calendar | Mondays every other week, 3:00pm-3:30pm PST <br>[Calendar](https://calendar.google.com/event?action=TEMPLATE&tmeid=Mm40OXM2NzRuaXY1Z3J2dTRnNDBiZzk0N3FfMjAyMTA0MjZUMjIwMDAwWiBjaHJpc3RpZXdpbHNvbkBnb29nbGUuY29t&tmsrc=christiewilson%40google.com&scp=ALL)|
+
+| &nbsp;                                                      | Facilitators      | Company      | Profile                                       |
+| --------------------------------------------------------    | ----------        | -------      | ---------------------------------------       |
+| <img width="30px" src="https://github.com/bobcatfish.png">  | Christie Wilson   | Google       | [bobcatfish](https://github.com/bobcatfish)   |
+| <img width="30px" src="https://github.com/skaegi.png">      | Simon Kaegi       | IBM          | [skaegi](https://github.com/skaegi)           |
+| <img width="30px" src="https://github.com/jerop.png">       | Jerop Kipruto     | Google       | [jerop](https://github.com/jerop)             |
+| <img width="30px" src="https://github.com/pierretasci.png"> | Pierre Tasci      | Apple        | [pierretasci](https://github.com/pierretasci) |
+| <img width="30px" src="https://github.com/jmcshane.png">    | James McShane     | SuperOrbital | [jmcshane](https://github.com/jmcshane)       |
```

Comment:
@skaegi @jerop @pierretasci @jmcshane I think in the survey you all indicated you were open to facilitating, but absolutely no problem if you've changed your mind (or let me know if I misread it!)

assuming you are okay with this, I've also added you into the [facilitator rotation](https://docs.google.com/document/d/17PodAxG8hV351fBhSu7Y_OIPhGTVgj6OJ2lPphYYRpU/edit#heading=h.av8xgx83mnem) in the agenda doc 🙏

Reply:
happy to facilitate 😄

Reply:
ditto!